src/modules/SharedModule/components/AuthContext/AuthContext.tsx

```diff
@@ -0,0 +1,33 @@
+
+import { jwtDecode } from 'jwt-decode'
+import { createContext, useEffect, useState } from 'react'
+
+     export const AuthContext = createContext(null);
+
+     export default function AuthContextProvider (props:any) {
+
+         const [loginData, setLoginData] = useState('');
+
+         let baseUrl = 'https://upskilling-egypt.com:3003/api/v1'
+
+         const saveLoginData = () => {
+             const encodedCode:any = localStorage.getItem('token');
+             const decodedCode:any = jwtDecode(encodedCode);
```

Comment:
We can call the get Current User api after login, and get all  user information  including user avatar  neeeded in the navbar from the  api.